libs/SalesforceReact/SalesforceReactTests/ReactHarnessTests.m

```diff
@@ -23,12 +22,26 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <UIKit/UIKit.h>
+#import "ReactTestCase.h"
 
-#import "SFPluginTestSuite.h"
+@interface ReactHarnessTests : ReactTestCase
```

Comment:
Tests to make sure the harness code works.
There are four tests:
- one that passes synchronously
- one that fails synchronously
- one that passes asynchronously
- one that fails asynchronously